coordinator/config.go

```diff
@@ -31,6 +34,23 @@ type Config struct {
 	LogSettings logger.Settings
 }
 
+func (c Config) IsValid() error {
+	if c.NumUsersInc <= 0 {
+		return errors.New("NumUsersInc should be greater than 0")
+	}
+	if c.NumUsersDec <= 0 {
+		return errors.New("NumUsersDec should be greater than 0")
+	}
+	if c.RestTimeSec <= 0 {
+		return errors.New("RestTimeSec should be greater than 0")
+	}
```

Comment:
Can we also include the values of NumUsersInc/NumUsersDec/RestTimeSec respectively in the error messages?

Reply:
Yep sounds good.